x-pack/plugins/apm/public/utils/__test__/url.test.tsx

```diff
@@ -132,9 +132,9 @@ function getUnconnectedKibanLink() {
 describe('UnconnectedKibanaLink', () => {
   it('should have correct url', () => {
     const wrapper = getUnconnectedKibanLink();
-    const href = wrapper.find('EuiLink').prop('href');
+    const href = wrapper.find('EuiLink').prop('href') || '';
```

Comment:
`|| ''` is due to typescript, right? I was having the same issue (thus `href!`). 
I think it's best to keep the typescript hacks in typescript land (if possible). 

Does `const href = wrapper.find('EuiLink').prop('href') as string` work? if not, just leave it as is.